k8s/jenkins/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,250 @@
 actions:
-- name: dummy test
+- name: Jenkins service login
   bashTest:
     script: |-
-      exit 0
+      SERVICE="${APP_INSTANCE_NAME}-jenkins-ui"
+      IP=''
+      PORT=''
+      PASSWORD=''
+      FAILED=0
+
+      function wait_for_service_ip {
+        IP=$(kubectl get service "${SERVICE}" \
+          --namespace "${NAMESPACE}" \
+          -ojsonpath="{.spec.clusterIP}")
+        if [[ $? -ne 0 ]]; then
+          IP=""
+        fi
+      }
+
+      function wait_for_service_port {
+        PORT=$(kubectl get service "${SERVICE}" \
+          --namespace "${NAMESPACE}" \
+          -ojsonpath="{.spec.ports[0].port}")
+        if [[ $? -ne 0 ]]; then
+          PORT=""
+        fi
+      }
+
+      function get_jenkins_pod {
```

Comment:
The k8s tools is waiting for `STS` and `POD` resources. Please remove this.

Reply:
It's for consistency with the rest of the code.